### PR TITLE
Chapter 08 - 0 08_0_Expandiendo_Bitcoin_Transacciones_Otros.md

### PR DESCRIPTION
**Pull request by javiervargas**

### CODE REVIEW

File: es/08_0_Expandiendo_Bitcoin_Transacciones_Otros.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Capítulo ocho: Expandiendo las transacciones de Bitcoin de otras formas
```
Comment (csralvall) [inline es/08_0_Expandiendo_Bitcoin_Transacciones_Otros.md:1]: - `Expandiendo las transacciones de Bitcoin en otros sentidos`
- Requiere modificar LEEME. #377

File: es/08_0_Expandiendo_Bitcoin_Transacciones_Otros.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Los objetivos de apoyo incluyen la capacidad de:
+
+   * Comprender los diferentes tipos de bloqueos de tiempo
+   * Plan para el poder de Locktime
+   * Plan para el poder de OP_RETURN
+
+## Tabla de contenido
+
+   * [Sección uno: Envío de una transacción con un tiempo de bloqueo](08_1_Enviando_una_Transaccion_con_BloqueodeTiempo.md)
```
Comment (csralvall) [inline es/08_0_Expandiendo_Bitcoin_Transacciones_Otros.md:20]: - `[Sección uno: Enviando una transacción con bloqueo de tiempo](08_1_Enviando_una_Transaccion_con_Bloqueo_de_Tiempo.md)`

File: es/08_0_Expandiendo_Bitcoin_Transacciones_Otros.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Capítulo ocho: Expandiendo las transacciones de Bitcoin de otras formas
+
+En la definición de transacciones básicas en el [Capítulo Seis](06_0_Expanding_Bitcoin_Transactions_Multisigs.md) se comentó que se enviaron _fondos_ _inmediatamente_, sin embargo ambos son elementos que se pueden cambiar. Esta sección final sobre la expansión de las transacciones de Bitcoin habla sobre cómo enviar cosas que no sean dinero en efectivo y cómo hacerlo en otro momento que no sea ahora.
```
[the remark this answers is not in the document]
Comment (javiervargas) [reply es/08_0_Expandiendo_Bitcoin_Transacciones_Otros.md:3]: This text is correct, I don't see your point.

File: es/08_0_Expandiendo_Bitcoin_Transacciones_Otros.md (hunk 1 above)
Comment (javiervargas) [reply es/08_0_Expandiendo_Bitcoin_Transacciones_Otros.md:1]: Please Could you precise what changes are required?,  and Where?

Comment (csralvall) [reply es/08_0_Expandiendo_Bitcoin_Transacciones_Otros.md:1]: Oh, this is OK. I'm just linking this pr with the LEEME pr to keep account of needed modifications for filenames.

**Comment by javiervargas** [comment]
@shannona ready to merge

**Review by csralvall — APPROVED** [review]

**Comment by shannona** [comment]
Thank you.

**Pull request merged**